internal/api/identity.go

```diff
@@ -65,3 +69,46 @@ func (a *API) DeleteIdentity(w http.ResponseWriter, r *http.Request) error {
 
 	return sendJSON(w, http.StatusOK, map[string]interface{}{})
 }
+
+func (a *API) LinkIdentity(w http.ResponseWriter, r *http.Request) error {
+	ctx := r.Context()
+	user := getUser(ctx)
+	rurl, err := a.GetExternalProviderRedirectURL(w, r, user)
+	if err != nil {
+		return err
+	}
+	skipHTTPRedirect, err := strconv.ParseBool(r.URL.Query().Get("skip_http_redirect"))
+	if err != nil {
+		skipHTTPRedirect = false
+	}
```

Comment:
You can just do this and not deal with parsing...

```suggestion
	skipHTTPRedirect := r.URL.Query().Get("skip_http_redirect") == "true"
```